src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -256,7 +267,13 @@ def __init__(self, config):
         self.ffn = FFN(config)
         self.output_layer_norm = nn.LayerNorm(normalized_shape=config.dim, eps=1e-12)
 
-    def forward(self, x, attn_mask=None, head_mask=None, output_attentions=False):
+    def forward(
+        self,
+        x: torch.Tensor,
+        attn_mask: Optional[torch.Tensor] = None,
+        head_mask: Optional[torch.Tensor] = None,
+        output_attentions: bool = False,
+    ) -> Tuple[torch.Tensor, ...]:
```

Comment:
```suggestion
    ) -> Tuple[torch.Tensor]:
```
I think `Tuple[torch.Tensor]` covers tuples of varying lengths!

Reply:
Interestingly it seems MyPy doesn't complain about the difference but PyLance does!

Reply:
Ah, okay. In that case my correction is wrong, keep the ellipsis!